### ISSUE TITLE
process out of memory

### ISSUE THREAD

**Issue by gabrielmuresan** [driftyco/ionic#8657]
live reloading when this happens.

```
[20:36:44]  bundle dev update started ...
FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - process out of memory

<--- Last few GCs --->
 2498941 ms: Scavenge 1397.6 (1457.3) -> 1397.6 (1457.3) MB, 3.8 / 0 ms (+ 2.9 ms in 1 steps since last GC) [allocation failure] [incremental marking delaying mark-sweep].
 2500205 ms: Mark-sweep 1397.6 (1457.3) -> 1395.9 (1457.3) MB, 1263.7 / 0 ms (+ 4.2 ms in 2 steps since start of marking, biggest step 2.9 ms) [last resort gc].
 2501344 ms: Mark-sweep 1395.9 (1457.3) -> 1397.3 (1457.3) MB, 1138.8 / 0 ms [last resort gc].
<--- JS stacktrace --->
==== JS stack trace =========================================
Security context: 0x1513a30b4629 <JS Object>
    1: decode(aka decode) [/Users/muresan/work_folder/drpciv-app-2/node_modules/rollup/dist/rollup.js:~302] [pc=0x3778a8397997] (this=0x1513a30041b9 <undefined>,mappings=0x2e8f57704101 <Very long string[2461888]>)
    2: render [/Users/muresan/work_folder/drpciv-app-2/node_modules/rollup/dist/rollup.js:8971] [pc=0x3778a83d9e0d] (this=0x2ea6dff71519 <a Bundle with map 0x245759756431>,options=0xa...

```

**Comment by okonon** [comment]
What version of `@ionic/app-scripts` you are running? try updating them

**Comment by jgw96** [comment]
Hello @gabrielmuresan ! Thanks for opening an issue with us! We have fixed this in the latest app-scripts release v0.0.30. To update you can run `npm install` in your app. Thanks!
